### ISSUE TITLE
attempt to index local 'frame' (a nil value)

### ISSUE THREAD

**Issue by burnep** [Verubato/framesort#33]
Happened in 3v3 when playing warrior / mage / druid

```Lua
1x FrameSort/WoW/Frame.lua:151: attempt to index local 'frame' (a nil value)
[string "@FrameSort/WoW/Frame.lua"]:151: in function `GetFrameUnit'
[string "@FrameSort/Modules/Sorting/SecureNoCombat.lua"]:17: in function <FrameSort/Modules/Sorting/SecureNoCombat.lua:15>
[string "=[C]"]: in function `sort'
[string "@FrameSort/Modules/Sorting/SecureNoCombat.lua"]:358: in function <FrameSort/Modules/Sorting/SecureNoCombat.lua:337>
[string "@FrameSort/Modules/Sorting/SecureNoCombat.lua"]:417: in function <FrameSort/Modules/Sorting/SecureNoCombat.lua:391>
[string "@FrameSort/Modules/Sorting/SecureNoCombat.lua"]:504: in function <FrameSort/Modules/Sorting/SecureNoCombat.lua:469>
[string "=(tail call)"]: ?
[string "@FrameSort/Modules/Sorting/Sorting.lua"]:45: in function `Run'
[string "@FrameSort/Modules/Runner.lua"]:54: in function `callback'
[string "@FrameSort/Scheduling/Scheduler.lua"]:61: in function `RunWhenCombatEnds'
[string "@FrameSort/Modules/Runner.lua"]:52: in function `Run'
[string "@FrameSort/Modules/Runner.lua"]:107: in function `callback'
[string "@FrameSort/Scheduling/Scheduler.lua"]:29: in function <FrameSort/Scheduling/Scheduler.lua:27>
[string "@FrameSort/Scheduling/Scheduler.lua"]:37: in function <FrameSort/Scheduling/Scheduler.lua:33>

Locals:
self = <table> {
 ToFrameChain = <function> defined @FrameSort/WoW/Frame.lua:58
 GetContainer = <function> defined @FrameSort/WoW/Frame.lua:218
 RaidFrames = <function> defined @FrameSort/WoW/Frame.lua:240
 GetFrameUnit = <function> defined @FrameSort/WoW/Frame.lua:150
 LayoutType = <table> {
 }
 ExtractGroups = <function> defined @FrameSort/WoW/Frame.lua:187
 IsFlat = <function> defined @FrameSort/WoW/Frame.lua:130
 PartyFrames = <function> defined @FrameSort/WoW/Frame.lua:233
 FramesFromChain = <function> defined @FrameSort/WoW/Frame.lua:113
 ContainerType = <table> {
 }
 ExtractUnitFrames = <function> defined @FrameSort/WoW/Frame.lua:162
 ArenaFrames = <function> defined @FrameSort/WoW/Frame.lua:247
 IsForbidden = <function> defined @FrameSort/WoW/Frame.lua:251
 GetFrames = <function> defined @FrameSort/WoW/Frame.lua:29
}
frame = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'frame' (a nil value)"

```

**Comment by Verubato** [comment]
Thanks for the bug report.

**Comment by Verubato** [comment]
Fixed in 6.2.2.
